operations/helm/charts/mimir-distributed/templates/_helpers.tpl

```diff
@@ -50,10 +50,20 @@ Create chart name and version as used by the chart label.
 {{- end -}}
 
 {{/*
-Calculate image name based on whether enterprise features are requested
+Build mimir image reference based on whether enterprise features are requested. The component local values always take precedence.
+Params:
+  ctx = . context
+  component = component name (optional)
```

Comment:
`componentSectionFromName` errors out if the component doesn't exist, so i think `component` isn't optional